### ISSUE TITLE
Autoddrive folder not showing

### ISSUE THREAD

**Issue by Gibbanator** [Courseplay/courseplay#6603]
Hey All,
Please excuse the post if been posted. I did a search and nothing came up for the answer I was looking for.

I had two autodrives showing on my map login so I decided to just delete one.. The one I deleted was the newest version so no biggie I just deleted both and add back the new version but now Autodrive folder is not showing up on Couseplay to pick routes anymore. All my routes for autodrive are still there just not available to pick in courseplay. running version6.03.0017 for courseplay and 1.1.0.7 for Autodrive.

Is there a way to get courseplay to read autodrive? I see nothing in settings to turn on or off or add anything for autodrive.

**Comment by Kobra299** [comment]
ok they will not reply to your question because autodrive is a different mod so you might want to try the [autodrive github](https://github.com/Stephan-S/FS19_AutoDrive)

**Comment by Gibbanator** [comment]
its a Courseplay issue why would I post it on auto drive? Auto drive is working fine just not showing on Courseplay.

**Comment by Tensuko** [comment]
Do you still have your AD network then ?

**Comment by Gibbanator** [comment]
just an update Tensuko.. I have updated to CP 6.03.0019 but there still is no folder for Autodrive in CP. Was working fine.. Than I did the autodrive delete and reinstall. I don't see why the folder would not be in CP when Autodrive still has all the courses for the map.

**Comment by Tensuko** [comment]
Did you try the reload button ?

**Comment by Gibbanator** [comment]
if your talking about the circular arrows where the courses are yes I have many times hoping it would work.

**Comment by Gibbanator** [comment]
Okay I got this to work. The auto drive version was the latest version from git hub 1.1.0.7 but there was the same version in the mod area of the game (1.1.0.7) but not showing installed. I uninstalled the Git hub mod and than rebooted the game and installed the one in the mod area (same version) Now the Auto drive folder is showing in the CP courses area.

I do not understand why that made a difference.. :(
